src/connectors/redbull.js

```diff
@@ -0,0 +1,136 @@
+ * eg. https://www.redbull.com/int-en/videos/episode-3-full-track-video-check-your-dms
+ */
+const PAGE_TITLE_REG_EXP = {
+	pattern: /(.+)\sby\s([^-–]+)/,
+	groups: { artist: 2, track: 1 },
+};
+
+let artistTrack = {
+	artist: null,
+	track: null,
+};
+
+Connector.playerSelector = '#app';
+
+Connector.currentTimeSelector = '.rbPlyr-timeElapsed';
+
+Connector.durationSelector = '.rbPlyr-timeDuration';
+
+Connector.getUniqueID = () => {
+	const text = Util.getAttrFromSelectors(
+		'[data-content-id]', 'data-content-id');
+	return text && text.split(':').slice(-2).shift();
+};
+
+Connector.getArtistTrack = () => {
+	for (const getter of ARTIST_TRACK_GETTERS) {
+		artistTrack = getArtistTrackFrom(getter);
+
+		if (!Util.isArtistTrackEmpty(artistTrack)) {
+			break;
+		}
+	}
+
+	return artistTrack;
+};
+
+Connector.playButtonSelector = 'rbPlyr-playPause:not(.rbPlyr-pause)';
+
+Connector.isScrobblingAllowed = () => {
+	const pageType = Util.getAttrFromSelectors(
+		'meta[property="og:type"]', 'content');
+
+	const categoryLink = Util.getAttrFromSelectors(
+		'a[href*="/tags/"]', 'href');
+
+	return pageType && ALLOWED_TYPES.includes(pageType) && categoryLink && 'music' === categoryLink.split('/').pop();
+};
+
+function getArtistTrackFrom(getter) {
+	// escape special characters for regExp
+	const track = artistTrack.track && artistTrack.track.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+
+	if (getter === 'videoTitle') {
+		// Example: https://www.redbull.com/int-en/episodes/red-bull-records-the-aces-daydream
+		const text = Util.getAttrFromSelectors(
+			'meta[property="og:title"]', 'content');
+
+		if (!text) {
+			return artistTrack;
+		}
+		artistTrack = Util.splitArtistTrack(text);
+
+		if (Util.isArtistTrackEmpty(artistTrack)) {
+			artistTrack = Util.processYtVideoTitle(text);
+		}
+
+		if (Util.isArtistTrackEmpty(artistTrack)) {
+			artistTrack.track = text;
+		}
+
+	} else if (getter === 'pageTitle') {
+		// Example: see Line 14
+		const pageTitle = Util.getTextFromSelectors('title');
+		const text = pageTitle && pageTitle
+			.split(':').pop()
+			.replace(/( music)? video$/i, '');
+
+		if (!text || !text.match(track)) {
+			return artistTrack;
+		}
+		const regPage = text.match(PAGE_TITLE_REG_EXP.pattern);
+
+		if (regPage) {
+			artistTrack.artist = regPage[
+				PAGE_TITLE_REG_EXP.groups.artist];
+		}
+
+		if (Util.isArtistTrackEmpty(artistTrack)) {
+			const ytResult = Util.processYtVideoTitle(text);
+
+			if (ytResult.track.match(track)) {
+				artistTrack.artist = ytResult.artist;
+			} else if (ytResult.artist.match(track)) {
+				artistTrack.artist = ytResult.track;
+			}
+		}
+
+	} else if (getter === 'description') {
+		// Example: https://www.redbull.com/int-en/videos/check-your-dms-s1-e2-full-track-video
+		const text = Util.getAttrFromSelectors(
+			'meta[property="og:description"]', 'content');
+
+		if (!text) {
+			return artistTrack;
+		}
+
+		const regDesc = text.match(new RegExp(`(.*)${track}(.*)`));
```

Comment:
This expression also includes a trailing dot which prevents recognizing the track. It should be removed.
![image](https://user-images.githubusercontent.com/1119267/99870497-7cb97980-2be4-11eb-89eb-69a3a99c348a.png)